modules/brainbrowser/ajax/minc.php

```diff
@@ -64,11 +64,21 @@
     $minc_path = getMincLocation() . $minc_file;
 }
 
-if (!empty($minc_file)) {
-    readfile($minc_path);
+if (!is_readable($minc_path)) {
+    if (!file_exists($minc_path)) {
+        error_log("ERROR: $minc_path exists in the DB but not in the file system");
+        http_response_code(500);
+        exit();
+    } else {
+        error_log(
+            "$minc_path was requested but is not readable. " .
+            'Possible permission error'
+        );
+        http_response_code(403);
```

Comment:
a 403 means that the user doesn't have permission to access the resource, not that the server couldn't retrieve it.

Reply:
The quote type also change between lines for no good reason in the message above.